src/sentry/models/commit.py

```diff
@@ -10,7 +10,7 @@
 )
 from sentry.utils.cache import memoize
 
-_fixes_re = re.compile(r'\bFixes\s+([A-Za-z0-9_\-\s\,]+)\b', re.I)
+_fixes_re = re.compile(r'\b(Fix|Fixes|Fixed|Close|Closes|Closed|Resolve|Resolves|Resolved)\s+([A-Za-z0-9_\-\s\,]+)\b', re.I)
```

Comment:
Just `(?:` for the group to avoid it being captured.

So like:

```python
r'\b(?:Fix|Fixes)\s...
```
This will be marginally more efficient and doesn't create the extra capture group that you don't need.